src/core/modules/impl/DefaultCreateContract.ts

```diff
@@ -71,7 +71,7 @@ export class DefaultCreateContract implements CreateContract {
         transfer
       });
     } else {
-      throw new Error('Unable to write Contract Source.');
+      throw new Error('Unable to write Contract Source.\n'+resource.statusText.msg);
```

Comment:
hmm. `resource`? did you mean `response`?
Also - the `statusText` is defined as a string (so there is no "msg" property):
```
post(transaction: Transaction | Buffer | string | object): Promise<{
        status: number;
        statusText: string;
        data: any;
    }>;
```

How about changing to 
```
`Unable to write Contract Source.\n${response?.statusText}`
```